core/i18n/certa.json

```diff
@@ -0,0 +1,25 @@
+{
+  // Comments are allowed here!
+  "testBundle": "./lib/cjs/test/webpack/bundled-tests.js",
+  "ports": {
+    "debugging": 5858,
+    "frontend": 3090,
+    "frontendDebugging": 9223
+  },
+  "mochaOptions": {
+    "timeout": 10000,
+    "reporter": "node_modules/@itwin/build-tools/mocha-reporter",
+    "reporterOptions": {
+      "mochaFile": "lib/test/junit_results.xml"
```

Comment:
i think you also need to update `common/config/azure-pipelines/templates/publish-test-results.yaml` so we pickup the test results

Reply:
Ah, I see. I didn't know about that. Added.

Also, I noticed there are other core packages that aren't listed in `publish-test-results.yaml`. I wouldn't add them in this PR, but was wondering if this is something that should get looked at?

Reply:
if we have tests and dont publish them then yes we should update the pipelines; i guess ppl are unaware of this step so it can get snuck past a pr reviewe